addons/web/static/src/js/views/form/form_controller.js

```diff
@@ -675,7 +675,7 @@ var FormController = BasicController.extend({
             on_saved: ev.data.on_saved,
             on_remove: ev.data.on_remove,
             readonly: ev.data.readonly,
-            deletable: ev.data.deletable,
+            deletable: ev.data.deletable && ev.data.originView !== 'list',
```

Comment:
it would be nice if `ev.data.deletable` already took the originView into account

Reply:
actually i think that it's already the case (with what you did l1717 relational_fields.js)